assets/js/googlesitekit/notifications/components/common/CTALinkSubtle.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { useDispatch } from 'googlesitekit-data';
+import { CORE_LOCATION } from '../../../datastore/location/constants';
+import useNotificationEvents from '../../hooks/useNotificationEvents';
+import { Button } from 'googlesitekit-components';
+import ExternalSVG from '../../../../../svg/icons/external.svg';
+
+export default function CTALinkSubtle( {
+	id,
+	ctaLink,
+	ctaLabel,
+	onCTAClick,
+	isCTALinkExternal = false,
+	gaConfirmEventLabel,
+} ) {
+	const trackEvents = useNotificationEvents( id );
+	const { navigateTo } = useDispatch( CORE_LOCATION );
+
+	const handleCTAClick = async ( event ) => {
+		if ( ! event.defaultPrevented ) {
+			event.preventDefault();
+		}
+
+		await onCTAClick?.( event );
+
+		await trackEvents.confirm( gaConfirmEventLabel );
+
+		if ( isCTALinkExternal ) {
+			global.open( ctaLink, '_blank' );
+		} else {
+			navigateTo( ctaLink );
+		}
+	};
```

Comment:
I'd vote to keep this natural, as `handleDismiss` in `assets/js/googlesitekit/notifications/components/common/Dismiss.js`. Using `global.open` in `RRMSetupSuccessSubtleNotification` was more of a hack to make the `core/forms` `setValues` action to work with `useRefocus`.

```suggestion
	const handleCTAClick = async ( event ) => {
		await onCTAClick?.( event );
		await trackEvents.confirm( gaConfirmEventLabel );
	};
```